packages/app/src/runner/event-manager.ts

```diff
@@ -315,6 +315,8 @@ export class EventManager {
       this.ws.emit('studio:save', saveInfo, (err) => {
         if (err) {
           this.reporterBus.emit('test:set:state', this.studioStore.saveError(err), noop)
+        } else {
+          this.studioStore.saveSuccess()
```

Comment:
I noticed that `cancel` explicitly calls `rerun` and `save` does not. Do you know if there's a reason? Or with your changes are we ok dropping the explicit `rerun`?

Reply:
The `save` scenario uses the file watching behavior to rerun the spec after the changes are saved to the file.